.github/workflows/ZygoteCI.yml

```diff
@@ -0,0 +1,30 @@
+name: ZygoteCI
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+
+jobs:
+  test:
+    name: Julia v${{ matrix.julia-version }} - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.0, 1.4]
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: ${{ matrix.julia-version }}
+          arch: x64
+      - uses: julia-actions/julia-buildpkg@latest
+      - name: Clone Zygote
+        uses: actions/checkout@v2
+        with:
+          repository: FluxML/Zygote.jl
+          path: "Zygote.jl"
+      - name: Run the tests
+        run: julia --project="Zygote.jl" -e "using Pkg; Pkg.develop(PackageSpec(path=\"./\")); Pkg.update(); Pkg.test()"
```

Comment:
this dev's the current branch of ChainRules, right?

Reply:
I think so, but to make sure, I'll introduce something that will definitely break Zygote and then if it fails revert before merging.